### DISCUSSION TITLE
Can Students Receive Free Swag Bags from GitHub Education?

### DISCUSSION THREAD

**Discussion by ahnayef** [community/community#55453] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey everyone,

I was wondering if you could help me out with something. I'm a student in the GitHub Education program and I'm really interested in getting one of those cool swag bags that they offer. But I'm not sure if they're only for teachers or if students can get them too.

Do you happen to know if GitHub Education offers swag bags to students? And if so, what do I need to do to get one? Any info you could provide would be awesome.

Thanks so much for your help. Looking forward to hearing back from you.

**Comment by Jordan99990** [comment, marked as answer]
Hi there! As far as I know, GitHub Education does offer swag bags to students who are enrolled in their program but the teacher has to get it for you I think. Also the availability, from what I understand, it really depends on your region you should receive more info about how to claim your swag bag if it is available in your region.

![image](https://github.com/community/community/assets/104220884/af244a81-58aa-4c5a-b041-5aa44cdedf4a)

**Reply by 20481A05K0** [reply]
Hello

**Comment by 20481A05K0** [comment]
Please tell me how should i get swag bag of github

**Reply by Rajputadit** [reply]
I'm Aditya Kumar Singh can i also get swag

**Comment by Prsanth** [comment]
How can I receive the bag

**Comment by Swatantra35** [comment]
yes
